### DISCUSSION TITLE
v2.1.0-BETA-2 Released - Install on your own risk

### DISCUSSION THREAD

**Discussion by Nisbo** [Nisbo/another_mvg#18] [Announcements]
* removed unnecessary entries from the translation files
* fixed an issue in the "big" card where the line break of the destination column was not correct formatted
* Added an option to change the format of the Departure time

![grafik](https://github.com/user-attachments/assets/5336a072-94df-4045-8322-adc1c99fb39b)

You have the following 3 options (the 1st one is the default option) 

- 16:27 +2 (16:29)
- 16:27 +2
- 16:29

In the example, 16:27 is the scheduled departure time, the train has a delay of 2 minutes, and 16:29 is the current departure time. If only the current departure time (Option 3) is displayed, the time will be shown in red if there is a delay.

Option 1 (same as before)

![grafik](https://github.com/user-attachments/assets/d9e111d7-65ec-42f6-b0e6-b6f0979b689f)

Option 2

![grafik](https://github.com/user-attachments/assets/e2149c9c-3eee-40b9-8e5b-acb3bdff3ba2)

Option 3
![grafik](https://github.com/user-attachments/assets/f77f8736-14d6-484c-bf1f-f5c79814d8ac)

<hr /><em>This discussion was created from the release <a href='https://github.com/Nisbo/another_mvg/releases/tag/v2.1.0.BETA.2'>v2.1.0-BETA-2 Released - Install on your own risk</a>.</em>

**Comment by JDTm** [comment]
Thanks for the Update! I am still looking for a tablet/Screen/etc to check it, before running to the train station.
